src/Turbo/README.md

```diff
@@ -0,0 +1,626 @@
+
+```php
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Mercure\PublisherInterface;use Symfony\Component\Mercure\Update;
+
+class MyController extends AbstractController
+{
+    public function publish(PublisherInterface $hub1): Response
+    {
+        $id = $hub1->publish(new Update('topic', 'content'));
+
+        return new Response("Update #{$id} published.");
+    }
+}
+```
+
+Changes made to entities marked with the `#[Broadcast]` attribute will be sent using all configured transport by default.
+You can specify the list of transports to use for a specific entity class using the `transports` parameter:
+
+```php
+<?php
+
+namespace App\Entity;
+
+use Symfony\UX\Turbo\Broadcast;
+
+#[Broadcast(transports: ['hub1', 'hub2'])]
+/** ... */
+class Book
+{
+    // ...
+}
+```
+
+Finally, generate the HTML attributes registering the Stimulus controller
+corresponding to your transport by passing an extra argument to `turbo_stream_listen()`:
+
+```twig
+<div id="messages" {{ turbo_stream_listen('App\Entity\Book', 'hub2') }}></div>
+```
+
+### Registering a Custom Transport
+
+If you prefer using another protocol than Mercure, you can create your custom transports:
+
+```php
+<?php
+
+namespace App\Turbo;
+
+use Symfony\UX\Turbo\Broadcast;
+use Symfony\UX\Turbo\Broadcaster\BroadcasterInterface;
+
+class Broadcaster implements BroadcasterInterface
+{
+    public function broadcast(object $entity, string $action): void
+    {
+        // This method will be called everytime an object marked with the #[Broadcast] attribute is changed
+        $attribute = (new \ReflectionClass($entity))->getAttributes(Broadcast::class)[0] ?? null;
+        // ...
+    }
+}
+```
+
+```php
+<?php
+
+namespace App\Turbo;
+
+use Symfony\UX\Turbo\Twig\TurboStreamListenRendererInterface;use Symfony\WebpackEncoreBundle\Twig\StimulusTwigExtension;use Twig\Environment;
+
+class TurboStreamListenRenderer implements TurboStreamListenRendererInterface
+{
+    public function __construct(
+        private StimulusTwigExtension $stimulusTwigExtension,
+    ) {}
+
+    /**
+     * @param string|object $topic
+     */
+    public function renderTurboStreamListen(Environment $env, $topic) : string{
+        $this->stimulusTwigExtension->renderStimulusController($env, 'your_stimulus_controller', [/* controller values such as topic */]);
+    }
+
+    public static function getDefaultIndexName(): string
+    {
+        // The transport name, used as key in the service locator
+        // Alternatively, configure it using the other supported methods: https://symfony.com/doc/current/service_container/service_subscribers_locators.html#indexing-the-collection-of-services
+        return 'my-transport';
+    }
+}
+```
+
+Because these classes implement the `BroadcasterInterface` and `TurboStreamListenRendererInterface` interfaces,
```

Comment:
I think we should explain to use the `turbo.renderer.stream_listen` and `turbo.broadcaster` tags (and *then* say that if your services are autoconfigured, this is done automatically).

Reply:
done